camera/views/camera_actions.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="camera_action" model="ir.actions.act_window">
+        <field name="name">Cameras</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">camera.camera</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <menuitem
+        id="camera_menu"
+        name="Camera"
+        action="camera_action"
+        parent="base.menu_ir_property"
+        groups="base.group_no_one"
+    />
```

Comment:
This menu addition is a good point. Not sure if it's the best place, anyway we could keep it here